pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2867 @@
+	Filter           *PodSandboxFilter `protobuf:"bytes,1,opt,name=filter" json:"filter,omitempty"`
+	XXX_unrecognized []byte            `json:"-"`
+}
+
+func (m *ListPodSandboxRequest) Reset()         { *m = ListPodSandboxRequest{} }
+func (m *ListPodSandboxRequest) String() string { return proto.CompactTextString(m) }
+func (*ListPodSandboxRequest) ProtoMessage()    {}
+
+func (m *ListPodSandboxRequest) GetFilter() *PodSandboxFilter {
+	if m != nil {
+		return m.Filter
+	}
+	return nil
+}
+
+// PodSandbox contains minimal information about a sandbox.
+type PodSandbox struct {
+	// The id of the PodSandbox
+	Id *string `protobuf:"bytes,1,opt,name=id" json:"id,omitempty"`
+	// The name of the PodSandbox
+	Name *string `protobuf:"bytes,2,opt,name=name" json:"name,omitempty"`
+	// The state of the PodSandbox
+	State *PodSandBoxState `protobuf:"varint,3,opt,name=state,enum=runtime.PodSandBoxState" json:"state,omitempty"`
+	// Creation timestamps of the sandbox
+	CreatedAt *int64 `protobuf:"varint,4,opt,name=created_at" json:"created_at,omitempty"`
+	// The labels of the PodSandbox
+	Labels           map[string]string `protobuf:"bytes,5,rep,name=labels" json:"labels,omitempty" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
+	XXX_unrecognized []byte            `json:"-"`
+}
+
+func (m *PodSandbox) Reset()         { *m = PodSandbox{} }
+func (m *PodSandbox) String() string { return proto.CompactTextString(m) }
+func (*PodSandbox) ProtoMessage()    {}
+
+func (m *PodSandbox) GetId() string {
+	if m != nil && m.Id != nil {
+		return *m.Id
+	}
+	return ""
+}
+
+func (m *PodSandbox) GetName() string {
+	if m != nil && m.Name != nil {
+		return *m.Name
+	}
+	return ""
+}
+
+func (m *PodSandbox) GetState() PodSandBoxState {
+	if m != nil && m.State != nil {
+		return *m.State
+	}
+	return PodSandBoxState_READY
+}
+
+func (m *PodSandbox) GetCreatedAt() int64 {
+	if m != nil && m.CreatedAt != nil {
+		return *m.CreatedAt
+	}
+	return 0
+}
+
+func (m *PodSandbox) GetLabels() map[string]string {
+	if m != nil {
+		return m.Labels
+	}
+	return nil
+}
+
+type ListPodSandboxResponse struct {
+	// List of PodSandbox
+	Items            []*PodSandbox `protobuf:"bytes,1,rep,name=items" json:"items,omitempty"`
+	XXX_unrecognized []byte        `json:"-"`
+}
+
+func (m *ListPodSandboxResponse) Reset()         { *m = ListPodSandboxResponse{} }
+func (m *ListPodSandboxResponse) String() string { return proto.CompactTextString(m) }
+func (*ListPodSandboxResponse) ProtoMessage()    {}
+
+func (m *ListPodSandboxResponse) GetItems() []*PodSandbox {
+	if m != nil {
+		return m.Items
+	}
+	return nil
+}
+
+// ImageSpec is an internal representation of an image.  Currently, it wraps the
+// value of a Container's Image field (e.g. imageName, imageName:tag, or
+// imageName:digest), but in the future it will include more detailed
+// information about the different image types.
+type ImageSpec struct {
```

Comment:
@vishh introducing OCI images (even from local tar balls) was exactly my point instead of just assuming a docker registry to pull docker images from. I commented in #7203 though.